server-common/src/main/java/org/apache/gravitino/server/authorization/expression/AuthorizationExpressionConverter.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.gravitino.server.authorization.expression;
+
+import org.apache.gravitino.server.authorization.MetadataFilterHelper;
+
+/** Convert the authorization expression to OGNL expression. */
+public class AuthorizationExpressionConverter {
+
+  private AuthorizationExpressionConverter() {}
+
+  /**
+   * Convert the authorization expression to OGNL expression
```

Comment:
What's the meaning of OGNL?

Reply:
OGNL is used to execute complex authorization logic. For example, if having either the load table or select_table permission grants access to the load table permission, OGNL can be used to call the authorize method of GravitinoAuthorizer multiple times and perform Boolean operations to obtain the final authorization result. (Refer to https://github.com/apache/gravitino/issues/6829)